libs/coin-bitcoin/src/js-getFeesForTransaction.ts

```diff
@@ -27,7 +27,7 @@ const getFeesForTransaction = async ({
       address: o.address,
       isChange: o.isChange,
       value: new BigNumber(o.value),
-      hash: undefined,
+      hash: "", //TODO: check that convert `undefined` to empty string is ok.
```

Comment:
I am not very sure about the impact. Can we just keep undefined for safety ?

Reply:
I just checked. It is very safe. the return object is only used  to calculate the fees and the fees are not related to the hash. this hash field is useless in this case.

Reply:
Thanks